modules/graphql_views/src/Plugin/Deriver/ViewDeriver.php

```diff
@@ -128,7 +128,7 @@ public function getDerivativeDefinitions($basePluginDefinition) {
         'view' => $viewId,
         'display' => $displayId,
         'paged' => $paged,
-        'cache_tags' => $view->getCacheTags(),
+        'cache_tags' => $view->getExecutable()->getCacheTags(),
```

Comment:
This happens in a different place. These are the cache tags for invalidating the SCHEMA cache. Not the RESULT cache.